.github/workflows/ci.yml

```diff
@@ -41,4 +41,65 @@ jobs:
       uses: actions/setup-go@v1
       with:
         go-version: ${{ secrets.GO_VERSION }}
-    - run: GOPROXY=direct GOSUMDB=off GO111MODULE=on go build .
\ No newline at end of file
+    - run: GOPROXY=direct GOSUMDB=off GO111MODULE=on go build .
+    - name: Upload artifacts
+      uses: actions/upload-artifact@v2
+      with:
+        name: meshery-consul
+        path: ./meshery-consul
+  e2e-tests:
+    needs: [golangci, build]
+    name: End-to-end tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Check out meshery code
```

Comment:
Is there any specific reason to use `meshery` to access `meshery-consul`?
locally, I just run `meshery-consul` and run tests directly against it (localhost:10002).

Reply:
Oh, there is no reason to have `meshery` so. I was thinking it was necessary to have it.
Let me remove that part.